certification/internal/engine/skopeo.go

```diff
@@ -0,0 +1,108 @@
+package engine
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os/exec"
+	"regexp"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/errors"
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/cli"
+	log "github.com/sirupsen/logrus"
+)
+
+func NewSkopeoEngine() *cli.SkopeoEngine {
+	var engine cli.SkopeoEngine = SkopeoEngine{}
+	return &engine
+}
+
+type SkopeoEngine struct{}
+
+func (e SkopeoEngine) ListTags(image string) (*cli.SkopeoListTagsReport, error) {
+
+	imageName, err := e.imageName(image)
```

Comment:
Does the call to `imageName` (and its subsequent functions) need to be part of the `SkopeoEngine` struct?  It seems that they don't use the struct's vars or funcs after the initial call to `imageName`.

Reply:
This method is used to strip digest/sha from the image name. Skopeo is the only engine that needs this functionality; but `imageName` method can be moved to util/container for the future.

Reply:
Thanks for clarifying!